src/main/ui/app/component/NamedList.jsx

```diff
@@ -27,9 +27,8 @@ export default class NamedList extends Component {
         const COLUMNS_IF_SIZE = 5;
 
         const list = this.props.list;
-        const colSize = list.addOns.length > COLUMNS_IF_SIZE ?
```

Comment:
@reubenvarghese1 okay, so I went back and recreated my old code for this and I posted here to ask people which works better: https://talk.openmrs.org/t/replacing-modulus-design-thoughts/9619/37

Looking at them side-by-size I now think that the 2-column view is misguided, since the eye has to move horizontally as well as vertically, and this is distracting. So, unless several people reply to my post saying they like the 2-column view, we should go with your solution. (And if we do this, also remove the COLUMNS_IF_SIZE constant, and also the half function earlier in this file.)

If we do end up going with the 2-column version then the code should look like

    render() {
        const COLUMNS_IF_SIZE = 10;

        const list = this.props.list;
        const colSize = list.addOns.length >= COLUMNS_IF_SIZE ?
                        "col-md-6 col-sm-12 col-xs-12" :
                        "col-md-12 col-sm-12 col-xs-12";

        return (
                <div>
                    <h1>{list.name}</h1>
                    <h3>{list.description}</h3>
                    { list.addOns.length >= COLUMNS_IF_SIZE ?
                      <div className="row">
                          <div className={colSize}>
                              { this.half(list.addOns, 0).map(addon => this.display(addon)) }
                          </div>
                          <div className={colSize}>
                              { this.half(list.addOns, 1).map(addon => this.display(addon)) }
                          </div>
                      </div>
                      :
                      <div className="row">
                          <div className={colSize}>
                              { list.addOns.map(addon => this.display(addon)) }
                          </div>
                      </div>
                    }
                </div>
        )
    }